server/agentProxy.go

```diff
@@ -42,6 +44,29 @@ func MakeAgentProxy(log slog.Instance, gpg etc.PGPInterface, tm etc.TokenManager
 	}
 }
 
+func injectUniquenessFields(log slog.Instance, json map[string]interface{}) {
+	uniqueString := ""
+	tries := 0
+	for len(uniqueString) == 0 && tries < maxUUIDTries {
+		u, err := uuid.NewRandom()
+		if err != nil {
+			log.Warn("Error generating UUID: %q. Trying again", err)
+			tries++
+			continue
+		}
+		uniqueString = u.String()
+	}
```

Comment:
We can write a less verbose `for`:
```suggestion
	var uniqueString string
	for tries := 0; tries < maxUUIDTries; tries++ {
		u, err := uuid.NewRandom()
		if err == nil {
		        uniqueString = u.String()
		        break
		}
		log.Warn("Error generating UUID: %q. Trying again", err)
	}
```

Reply:
Nice!